backend/trafficrules/src/test/java/com/coffeejug/trafficrules/service/StatisticServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package com.coffeejug.trafficrules.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+class StatisticServiceTest {
+
+    @Mock
+    private StatisticService statisticService;
+
+    @Test
+    void shouldReturnZero() {
+
+        Long start = statisticService.getUptime();
+        assertThat(start).isEqualTo(0L);
+        Long end = null;
+        try {
+            Thread.sleep(1000);
+            end = statisticService.getUptime();
+        } catch (InterruptedException e) {
+            e.printStackTrace();
```

Comment:
In case of an exception, the test won't fail. It's better to declare the method to throw it

Reply:
In case of exeption end==null and will crash here         assertThat(end).isEqualTo(0L);

Reply:
But it's better to know why the test failed. If it failed because of the exception we need to see it.

Reply:
@sergiykatriuk and this one here :)

Reply:
Haven't reached this yet.

Reply:
This can be a simple unit test, IMO. But even if you think that we need this to be an international test then we don't need to use mockito extension

Reply:
@sergiykatriuk is this resolved? I'm still convinced this should be a unit and not integrational test